src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -14,7 +14,6 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Component\HttpFoundation\Session\Session;
-use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
```

Comment:
This is used in a comment.
https://github.com/stloyd/symfony/blob/feature/drop_depr/src/Symfony/Bridge/Twig/AppVariable.php#L56